.github/workflows/release.yaml

```diff
@@ -10,14 +10,16 @@ concurrency: ${{ github.workflow }}-${{ github.ref }}
 
 jobs:
   release:
-    name: "Release"
     runs-on: ubuntu-latest
     environment: npm-deploy
     permissions: write-all
-    # Ensure that the workflow runs only after a commit is pushed into the branch
+    # First check ensures that the workflow runs only when the changesets PR commit is pushed into the branch.
+    # Second check ensures that the workflow runs only after a commit is pushed into the branch,
     # and not when the branch is created.
     # This is to avoid running the workflow when a release/* branch is created.
-    if: github.event.before != '0000000000000000000000000000000000000000'
+    if: |
```

Comment:
Now, because the workflow won't run at all except when the changesets PR gets merged, we won't be getting the manual approval requests in our notifications.